modules/javafx.graphics/src/main/java/com/sun/glass/ui/headless/HeadlessWindow.java

```diff
@@ -24,20 +47,14 @@ public class HeadlessWindow extends Window {
     private int maxWidth = -1;
     private int maxHeight = -1;
     private int originalX, originalY, originalWidth, originalHeight;
-
-    private boolean resizable;
     private boolean visible;
-    private boolean isFocusable;
-    private boolean enabled;
-    private boolean closed;
-    private float bg_r, bg_g, bg_b;
-    private float alpha;
-    private Pixels icon;
-    private Cursor cursor;
+
     private final ByteBuffer frameBuffer;
     private HeadlessView currentView;
     private HeadlessRobot robot;
 
+    private final int stride = 1000;
```

Comment:
I see you've created [JDK-8364201](https://bugs.openjdk.org/browse/JDK-8364201) for the configuration options, thank you.